plugins/fileservices/fileservices.cpp

```diff
@@ -60,6 +60,8 @@ static const char * EclDefinition =
 "export integer4 RECFMVB_RECSIZE := -1; // special value for SprayFixed record size \n"
 "export integer4 PREFIX_VARIABLE_RECSIZE := -3; // special value for SprayFixed record size \n"
 "export integer4 PREFIX_VARIABLE_BIGENDIAN_RECSIZE := -4; // special value for SprayFixed record size \n"
+"export FsDropZone := string{maxlength(255)}; \n"
```

Comment:
is the maxlength necessary? ... and a path _could_ be longer

Reply:
I saw this value at the fsLogicalFileName and I thought the drop zone path has some similarity with logical file name. So that's why I use it.

Reply:
default linux filename length is 255.
But this is a path, and the default is 4096, so 255 may well be too small  in some cases, so I think it should be changed.

However, is it  (and the other uses of maxlength in fileservices) actually needs/helpful at all. @ghalliday ?

Reply:
I would remove it (and just use a type of string in the record - it was needed when the original code was written, but maxlength has not been needed for many years.